File: python/tests/leetcode_test.py
import unittest
from python.leetcode import *

class LeetcodeTestCase(unittest.TestCase):

    def testSmallestEvenMultiple(self):
        self.assertEqual(10, smallestEvenMultiple(5))

    def testMinHeightShelves(self):
        _input = [[1,1],[2,3],[2,3],[1,1],[1,1],[1,1],[1,2]]
        self.assertEqual(6, minHeightShelves(_input, 4))

    def testSortPeople(self):
        names = ["Mary","John","Emma"];
        heights = [180,165,170]
        expected = ["Mary","Emma","John"]
        self.assertListEqual(expected, sortPeople(names, heights))


if __name__ == '__main__':
    unittest.main()

File: java/src/test/java/com/qingsong/leetcode/LeetcodeTest.java
package com.qingsong.leetcode;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;



/**
 * Unit test for simple Leetcode.
 */
public class LeetcodeTest {
    /**
     * Rigorous Test :-)
     */
    @Test
    public void testSmallestEvenMultiple() {
        Problem2413 problem = new Problem2413();
        Assertions.assertEquals(10, problem.smallestEvenMultiple(5));
    }

    @Test
    public void testMinHeightShelves() {
        Problem1105 problem = new Problem1105();
        int[][] input = {{1, 1,}, {2, 3}, {2, 3}, {1, 1}, {1, 1}, {1, 1}, {1, 2}};
        Assertions.assertEquals(6, problem.minHeightShelves(input, 4));
    }

    @Test
    public void testSortPeople() {
        Problem2418 problem = new Problem2418();
        String[] names = {"Mary","John","Emma"};
        int[] heights = {180,165,170};
        String[] expected = {"Mary","Emma","John"};
        Assertions.assertArrayEquals(expected, problem.sortPeople(names, heights));
    }
}
